packages/cli/src/commands/versions/lint.ts

```diff
@@ -27,8 +27,8 @@ export const includedFilter = (name: string) =>
 
 // Packages that are not allowed to have any duplicates
 const FORBID_DUPLICATES = [
-  /^@backstage\/core$/,
-  /^@backstage\/core-api$/,
+  /^@backstage\/core-app-api$/,
+  /^@backstage\/core-plugin-api$/,
   /^@backstage\/plugin-/,
```

Comment:
Duplicates of `@backstage/plugin-catalog-react` is allowed as well. Not really part of this change but if you're here anyway 😅

Reply:
Alright, added!